tests/const_dsm_5.py

```diff
@@ -3,10 +3,471 @@
 from .const import SERIAL, SID, UNIQUE_KEY  # pylint: disable=unused-import
 
 # DSM 5 RAW DATA
-DSM_5_LOGIN = {}
+DSM_5_LOGIN = {
+    {'data': {'sid': SID}, 'success': True}
+}
 
-DSM_5_INFORMATION = {}
+DSM_5_INFORMATION = {
+    # Request failed
```

Comment:
We are gonna need this.

After https://github.com/ProtoThis/python-synology/issues/26#issuecomment-612647726 get answered, I will fix the lib so this API will work